app/lib/features/onboarding/widgets/missing_encryption_backup_widget.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:acter/common/themes/colors/color_scheme.dart';
+import 'package:acter/common/toolkit/buttons/primary_action_button.dart';
+import 'package:acter/features/onboarding/widgets/expect_decryption_failures_widget.dart';
+import 'package:acter/l10n/generated/l10n.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+
+class MissingEncryptionBackupWidget extends ConsumerWidget {
```

Comment:
This should be managed as Page as explained above.
```suggestion
class MissingEncryptionBackupPage extends ConsumerWidget {
```

Reply:
ok will add as a page